resources/views/includes/structure/breadcrumb.blade.php

```diff
@@ -1,3 +1,7 @@
 <div class="{{ isset($class) ? $class : 'col-md-12 shadow-sm p-3 mx-auto mb-3' }}">
-    {{ Breadcrumbs::render() }}
+    @if(!empty($page))
+        @include('components.breadcrumbs')
```

Comment:
Can you please add a brief comment on why this change was necessary? It looks to me like the breadcrumbs previously worked with nested pages, so I can't see any obvious reason that this would be necessary (although I'm sure I'm missing something):

https://fitzmuseum.cam.ac.uk/commercial-services/venue-hire

Reply:
Sure - This has been added because the client wants the case of each title to be true to the title input in Directus, which is tricky to achieve with the Breadcrumbs laravel plugin that handles the rest of the breadcrumbs on the site

Reply:
Thanks, understood 👍 

Can you please add the comment in the blade file :) ? Sorry I could have been clearer about that in my first message!